src/components/app/App.js

```diff
@@ -1,14 +1,14 @@
-import React from 'react';
+import React, {Fragment} from 'react';
 import Header from './header/Header';
 import Content from './content/Content';
```

Comment:
There is an approach in the React Style Guide to get rid of repeatable naming in the imports - by following component naming best practices:
https://github.com/airbnb/javascript/tree/master/react#naming
![image](https://user-images.githubusercontent.com/52723192/103790053-c73a6c80-5051-11eb-9025-c3665d40a269.png)

Please rename component and css files correspondingly.